src/core/objects/object_mover.h

```diff
@@ -52,6 +52,15 @@ class ObjectMover
 	/** Sets the start position. */
 	void setStartPos(const MapCoordF& start_pos);
 	
+	/** Sets corner point tolerance. A curve anchor point is considered a corner
+	 * point during editing when the moving curve handle is more than
+	 * corner_tolerance away from the direction line set by the opposite handle.
+	 * 
+	 * @param corner_tolerance Maximum difference in vector directions
+	 *                         in millimeters.
+	 */
+	void setCornerTolerance(const qreal corner_tolerance);
```

Comment:
`const` is fine in the definition, but has no effect and shall be omitted in the declaration.

Reply:
Ack.